experimental/dds/tree2/src/test/feature-libraries/modular-schema/modularChangeFamily.spec.ts

```diff
@@ -325,13 +328,39 @@ describe("ModularChangeFamily", () => {
 		};
 
 		it("prioritizes earlier build entries when faced with duplicates", () => {
+			const cursor = singleJsonCursor(1);
+			const chunk = uncompressedEncode(chunkTree(cursor, defaultChunkPolicy).cursor());
 			const change1: ModularChangeset = {
 				fieldChanges: new Map(),
-				builds: new Map([[undefined, new Map([[brand(0), singleJsonCursor(1)]])]]),
+				builds: new Map([
+					[
+						undefined,
+						new Map([
+							[
+								brand(0),
+								uncompressedEncode(
+									chunkTree(singleJsonCursor(1), defaultChunkPolicy).cursor(),
+								),
```

Comment:
this logic is a bit repetitive. Might be nice to (just locally in this test) make a Json -> encoded chunk function and call it in these three places.

Reply:
Cleaned up tests using a function